### ISSUE TITLE
Charlatano master

### ISSUE THREAD

**Issue by JagodekSTREAM** [Jire/Charlatano#800]
Using charlatano master branch is safe just like normal release?

**Comment by phooooon** [comment]
this hub is dead

**Comment by trolloos** [comment]
this one is more up to date:
https://github.com/AStupidRat/RatPoison

**Comment by phooooon** [comment]
@trollos is that one still U/d??

**Comment by trolloos** [comment]
you need to change the cmd window title for it to be ud

**Comment by phooooon** [comment]
Is this a joke

**Comment by trolloos** [comment]
nah man this is my .bat file:

> @echo off
		title %random%%random%%random%%random%%random%%random%%random%%random%%random%%random%%random%%random%
		echo Only continue once you are in game and can see your gun.
		echo.
		pause
		cd /d "%~dp0"
		set name=%random%%random%%random%%random%
		ren "*.jar" "%name%.jar"
		cls
		set args=-Xmx999M -Xms999M
		java %args% -jar "%name%.jar"
		pause

**Comment by phooooon** [comment]
May I ask, whats the actual use of changing the bat name?

**Comment by phooooon** [comment]
Just curious

**Comment by JagodekSTREAM** [comment]
@trolloos this serious works and is ud?

**Comment by trolloos** [comment]
VAC can detect window names like "cheat engine" or "charlatano" and they can ban you for it if they want or just flag your account for example. Of course it will not always happen but vac works in mysterious ways.

**Comment by JagodekSTREAM** [comment]
Yeah cuz I am using charlatano for 6 months and still without vac

**Comment by denizdeni** [comment]
@trolloos don't be a stupid, valve can't check window names it's so stupidly.

I can't believe I wrote that but if valve check the window names, almostly all cheaters get ban from games. because you MUST open program(injector, external or whatever).
